src/particleContainer/AutoPasContainer.h

```diff
@@ -17,7 +17,7 @@ class AutoPasContainer : public ParticleContainer {
 public:
 	AutoPasContainer();
 
-	~AutoPasContainer() override = default;
+	~AutoPasContainer() override {};
```

Comment:
_logFile should be closed again
```suggestion
	~AutoPasContainer() override {
#ifdef ENABLE_MPI
		_logFile.close()
#endif
	};
```